src/components/Form/CrossPodQueryForm.jsx

```diff
@@ -76,42 +82,42 @@ const CrossPodQueryForm = () => {
     }
   };
 
-  const formRowStyle = {
-    margin: '20px 0'
-  };
-
-  /* eslint-disable jsx-a11y/label-has-associated-control */
   return (
     <FormSection
       title="Cross Pod Search"
       state={state}
       statusType="Search status"
       defaultMessage="To be searched..."
     >
-      <form onSubmit={handleCrossPodQuery} autoComplete="off">
-        <div style={formRowStyle}>
-          <label htmlFor="cross-search-doc">Search document from username: </label>
-          <br />
-          <br />
-          <input
-            id="cross-search-doc"
-            size="25"
-            name="crossPodQuery"
-            {...username}
-            placeholder={selectedUser}
-          />
-        </div>
-        <div style={formRowStyle}>
-          <label htmlFor="cross-search-doctype">Select document type to search: </label>
-          <DocumentSelection htmlId="cross-search-doctype" />{' '}
-          <button disabled={state.processing} type="submit">
-            Search Pod
-          </button>
-        </div>
-      </form>
+      <Box display="flex" justifyContent="center" sx={{ minWidth: 120 }}>
+        <form onSubmit={handleCrossPodQuery} autoComplete="off">
+          <FormControl variant="standard" fullWidth>
+            <TextField
+              id="cross-search-doc"
+              name="crossPodQuery"
+              {...username}
+              placeholder={selectedUser}
+              label="Enter username"
+              InputProps={{
+                type: 'search'
+              }}
+              required
+            />
+          </FormControl>
+          <InputLabel id="cross-search-doctype">
+            <em>Select Document Type to Search: </em>
```

Comment:
Just got reminded of it when looking at the site again. Consider the section is pretty clearly defined about what it does (and since we're considering shortening phrases like "Document located at" to "Document found"), I think we can shorten this to just "Select Document Type".

The same could be said for Delete Document.
![Screen Shot 2023-05-28 at 1 04 24](https://github.com/codeforpdx/PASS/assets/14917816/52ecf2a5-dbc6-4ead-8d3a-02b563702ef9)